pom.xml

```diff
@@ -65,6 +65,7 @@
     <nuxeo.common.version>11.1-SNAPSHOT</nuxeo.common.version>
     <nuxeo.runtime.version>11.1-SNAPSHOT</nuxeo.runtime.version>
     <nuxeo.core.version>11.1-SNAPSHOT</nuxeo.core.version>
+    <nuxeo.distribution.version>11.1-SNAPSHOT</nuxeo.distribution.version>
```

Comment:
Is it still useful? Seems unused now.

Reply:
This property is still used by external packages.